_events/OSG-User-School-2023.md

```diff
@@ -18,14 +18,17 @@ header_image: "/images/Madison_Skyline.jpg"
 
 The OSG User School will occur on August 7–11, where you will learn to use high-throughput computing (HTC) systems to run large-scale computing applications at your campus or using the national-scale OSG.
 
-The School is ideal for researchers in any research area where large-scale computing is a vital part of the research process, anyone, especially students and staff, who supports researchers and are current or potential users of high-throughput computing, and instructors who teach future researchers and see value in integrating high-throughput computing into their curriculum.
-
 Applications are now open. The deadline for applications is Monday, 17 April 2023. Submit your application soon to ensure consideration!
 
 To apply, complete the [online application form](https://uwmadison.co1.qualtrics.com/jfe/form/SV_9shkUL5ZDSghKvk) and have someone email a letter of recommendation for you to [user-school@osg-htc.org](mailto:user-school@osg-htc.org).
 
 For more information about the School and to stay up-to-date with any announcements, please visit the [OSG User School 2023 site](https://osg-htc.org/user-school-2023/).
 
+To learn more about the event, check out these articles written about the OSG User School 2022:
```

Comment:
Instead of just a list of links, can each one have a sentence or so that might actually draw people in?